controllers/utils/s3.go

```diff
@@ -10,8 +10,22 @@ import (
 const (
 	RamenHubNamespace  = "openshift-dr-system"
 	BucketGenerateName = "odrbucket"
+
+	// s3
+	S3ProfilePrefix    = "s3profile"
+	S3Endpoint         = "s3CompatibleEndpoint"
+	S3BucketName       = "s3Bucket"
+	S3ProfileName      = "s3ProfileName"
+	S3Region           = "s3Region"
+	AwsAccessKeyId     = "AWS_ACCESS_KEY_ID"
+	AwsSecretAccessKey = "AWS_SECRET_ACCESS_KEY"
+
+	// ramen
+	RamenHubOperatorConfigName = "ramen-hub-operator-config"
 )
 
+var OriginMap = map[string]string{"RookOrigin": "rook", "S3Origin": "S3"}
```

Comment:
OriginMap map is common for all types of secrets, here rook is one type and s3 is one type. So, it is better keep this map in common place.

Reply:
moved to utils/secret.go since it is common to all secrets